src/data-base/SequelizeConnection.js

```diff
@@ -0,0 +1,10 @@
+const Sequelize = require("sequelize");
+
+module.exports = function(config) {
```

Comment:
you could use destructuring to manage the function params in a more clear way, or even add default values:
```javascript
function ({ database, user, password, connectionConfig } = {}) {
  . . .
}
```

Reply:
the change was made according to the suggestion for you example